### PR TITLE
💫 pre-compiled real-time ray-tracing with linear-gradients

### PR DESCRIPTION
**Pull request by PickleNik**
new intro animation:

https://github.com/typehero/typehero/assets/31113245/ee035baf-5d84-4e18-a0fa-9c04fe19f024

nav logo(before/after):
Screenshot 2023-10-17 at 1 59 28 AM
Screenshot 2023-10-17 at 1 59 19 AM

main logo(before/after)
Screenshot 2023-10-17 at 2 00 21 AM
Screenshot 2023-10-17 at 2 00 30 AM

other edge cases (before/after):
Screenshot 2023-10-17 at 1 57 38 AM
Screenshot 2023-10-17 at 1 57 31 AM

### CODE REVIEW

**Comment by PickleNik** [comment]
Screenshot 2023-10-17 at 2 40 17 AM
oh pluh, I've been tryna resolve checks not passing after rebasing to main

File: apps/og-image/app/components/logo3d.tsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ export function Logo3d() {
           fill="#3178C6"
         />
         <path
-          fillRule="evenodd"
-          clipRule="evenodd"
-          d="M261.441 348.723H333.158V299.842H133.263V348.723H204.629V566.368H261.441V348.723Z"
+          fill-rule="evenodd"
+          clip-rule="evenodd"
```
Comment (QuiiBz) [inline apps/og-image/app/components/logo3d.tsx:17]: Why?

Comment (bautistaaa) [reply apps/og-image/app/components/logo3d.tsx:17]: @PickleNik it has to be camelcase in react! `fillRule` `clipRule`

Comment (PickleNik) [reply apps/og-image/app/components/logo3d.tsx:17]: automatically copypasted from figma, fixed

**Review by bautistaaa — APPROVED** [review]

**Pull request merged**